genappbuilder/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example.genappbuilder</groupId>
+  <artifactId>genappbuilder-snippets</artifactId>
+  <packaging>jar</packaging>
+  <name>Generative AI App Builder Snippets</name>
+  <url>https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/genappbuilder</url>
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.2.0</version>
+  </parent>
+
+  <properties>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+
+  <!-- [START genappbuilder_install_with_bom] -->
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>libraries-bom</artifactId>
+        <version>26.13.0</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-discoveryengine</artifactId>
```

Comment:
The latest libraries bom only imports: `0.11.0` of the discoveryengine client. So I suggest you add the version explicitly here and add a comment to remove it when the `libraries-bom` has updated the version.

`<version>0.12.0</version>`

Reply:
That's interesting. It also didn't work when I added the version explicitly.
Does libraries-bom have access to the 0.12.0 version? It was just published recently

Reply:
libraries-bom will automatically include it in the next release. But we are not sure when the cloud sdk team will publish the next release. SO until then we can explicitly force to use `0.12.0` by adding the version tag